### ISSUE TITLE
jremillard/images-to-osm (111169572)

### ISSUE THREAD

**Issue by gfleetwood** [gfleetwood/asteres#1715]
https://github.com/jremillard/images-to-osm

Use Mask R-CNN/Keras/TensorFlow and OSM to find features in satellite images for fun.
